src/hooks.ts

```diff
@@ -139,6 +139,7 @@ export type ConstraintTypes = 'PointToPoint' | 'ConeTwist' | 'Distance' | 'Lock'
 
 export interface ConstraintOptns {
   maxForce?: number
+  maxMultiplier?: number
```

Comment:
Is `maxMultiplier` a good name for this? The inspiration comes from https://github.com/pmndrs/cannon-es/blob/6f11e268d5e196f7f4f5908dbf60ae6365bd6ab6/examples/tear.html#L63-L66

Reply:
I'll continue with assumption this name is acceptable.